pkg/kubelet/metrics/metrics.go

```diff
@@ -25,43 +25,74 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 )
 
-const kubeletSubsystem = "kubelet"
+const (
+	KubeletSubsystem              = "kubelet"
+	PodWorkerKey                  = "pod_worker_latency_microseconds"
```

Comment:
PodWorkerLatencyKey (and the same below). In the future we might want to add some other pod worker related metrics.